crates/primitives/src/receipt.rs

```diff
@@ -457,7 +369,9 @@ impl<'a> From<&'a Receipt> for ReceiptWithBloomRef<'a> {
     }
 }
 
-struct ReceiptWithBloomEncoder<'a> {
+/// Encoder for receipts with bloom.
+#[derive(Debug)]
+pub struct ReceiptWithBloomEncoder<'a> {
     bloom: &'a Bloom,
     receipt: &'a Receipt,
 }
```

Comment:
can we just remove this struct now given that encoding is implemented for both `ReceiptWithBloom<Receipt>` and `ReceiptWithBloom<&Receipt>`